<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverridePrompt</class>
 <widget class="QDialog" name="OverridePrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OverridePromptTitle</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 255); font-weight:bold;</string>
       </property>
       <property name="text">
        <string>PositionLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 170, 255); font-weight:bold;</string>
       </property>
       <property name="text">
        <string>AlreadyExists</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="fileName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="position">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 170, 255); font-weight:bold;</string>
     </property>
     <property name="text">
      <string>MakeYourChoice</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="button0">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>CopyAndReplace</string>
     </property>
     <property name="description">
      <string>CopyAndReplaceDescription</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="button1">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>DonotCopy</string>
     </property>
     <property name="description">
      <string>DonotCopyDescription</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="button2">
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="text">
      <string>CopyAndKeep2Files</string>
     </property>
     <property name="description">
      <string>CopyAndKeep2FilesDescription</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="remember">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>RememberMyChoice</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
